katello/katello.spec

```diff
@@ -26,6 +26,7 @@ Source9:    qpid-core-dump
 Source10:   katello-clean-empty-puppet-environments
 Source11:   katello-change-hostname
 Source12:   katello-repository-publish-check
+Source13:   katello-change-hostname.8.asciidoc
```

Comment:
This only adds it as a source and won't ensure its deployed properly to the system.

Reply:
@ehelms updated the man page